src/panels/config/script/ha-script-picker.ts

```diff
@@ -135,7 +135,7 @@ class HaScriptPicker extends LitElement {
         back-path="/config"
         .route=${this.route}
         .tabs=${configSections.automation}
-        .columns=${this._columns(this.hass.language)}
+        .columns=${this._columns(this.hass.locale.language)}
```

Comment:
```suggestion
        .columns=${this._columns(this.hass.locale)}
```

Reply:
`locale` is used for the format of the last triggered date

Reply:
Yes it is, but it gets that direction from `this.hass.locale` without needing to pass anything in. I reverted the change as part of using the backwards-compatible `hass.language` to simplify the PR review process. I don't think this needs to pass anything in to the `this._columns()` function?